packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -1330,7 +1329,7 @@ String? _validatePlugin(Plugin plugin, String platformKey, {
     if (_hasPluginInlineImpl(plugin, platformKey, pluginType: pluginType)) {
       return 'Plugin ${plugin.name}:$platformKey which provides an inline implementation '
           'cannot also reference a default implementation for $defaultImplPluginName. '
-          'Ask the maintainers of ${plugin.name} to either remove the implementation via `platforms: $platformKey: ${pluginType == _PluginType.dart ? 'dartPluginClass' : 'pluginClass'}` '
+          'Ask the maintainers of ${plugin.name} to either remove the implementation via `platforms: $platformKey: ${pluginType == _PluginType.dart ? 'dartPluginClass' : '[dart]pluginClass'}` '
```

Comment:
`dartpluginClass` wouldn't be correct, so the message here is misleading. It should say "pluginClass or dartPLuginClass".

Reply:
Thx. Done in d621a904e4a2ae069a1d02926ce7142553ff6fe4